x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/AbstractRemoteClusterSecurityTestCase.java

```diff
@@ -114,24 +115,19 @@ protected Settings restClientSettings() {
         return Settings.builder().put(ThreadContext.PREFIX + ".Authorization", token).build();
     }
 
-    protected static Map<String, Object> createCrossClusterAccessApiKey(String indicesPrivilegesJson) {
+    protected static Map<String, Object> createCrossClusterAccessApiKey(String roleDescriptorsJson) {
         initFulfillingClusterClient();
-        return createCrossClusterAccessApiKey(fulfillingClusterClient, indicesPrivilegesJson);
+        return createCrossClusterAccessApiKey(fulfillingClusterClient, roleDescriptorsJson);
     }
 
-    static Map<String, Object> createCrossClusterAccessApiKey(RestClient targetClusterClient, String indicesPrivilegesJson) {
+    static Map<String, Object> createCrossClusterAccessApiKey(RestClient targetClusterClient, String roleDescriptorsJson) {
```

Comment:
This change is needed in order to be able to specify more than one role so that we can test that multiple roles with DLS/FLS are treated as union when defined in the same role descriptor.